### PR TITLE
PRs for v1.1.0 release

### PR DESCRIPTION
**Pull request by Atticus29**
# Description

Adding all PRs in since the previous release PR #437 .
Includes:

93c9bf0f	dex 1341 - add collaboration from user page (#464)	464	1341
ad726aed	Dex 1435 new notifications to fe (#463)	463	1435
d558ff2f	Dex 1452 user preferences all merge (#462)	462	1452 and 1451
73b31977	Dex 1328 user manager view collabs (#460)	460	1328
fd9d326d	add a second empty encounter if the AGS submission has more than one … (#440)	440	759
343ad79a	DEX-1443 cherry pick e8c4a2d from DEX-1443-empty-dropdown-as-array be… (#459)	459	1443
f93ff9d6	dex 1462 - Fix edit custom field form's required toggle (#461)	461	1462
a29d497c	DEX-1445 add info alert on splash setting and general settings pages to better guide end user to enter URLs correctly. Form validation improvements forthcoming in subesequent ticket (#458)	458	1445
4aa18da0	dex 1417 - require both sighting time fields when reporting sighting (#456)	456	1417
b3a46e83	DEX-1433 remove EDM build date and commit hash from Site Status page (#455)	455	1433
764fcfdf	Integrate new site settings API (#449)	449	1379
c45eec8d	dex-1329 - show collaboration type in table (#451)	451	1329
d37b7edc	dex-1401 - incorrect sighting metadata payload (#453)	453	1401
a0539ff5	Fix page error when no notification message (#452)	452	N/A
a5b12209	Remove unused file PipelineStepProgressGrid (#450)	450	N/A
c845095f	Proposed table action group refactor (#448)	448	N/A
092f4693	Remove unused relationships code (including relationships custom field) (#446)	446	N/A
82c96d8a	dex-1017 - individual photo gallery - fix copy (#447)	447	1017
a5b1412f	dex-1343 - clear collaboration failure after success (#445)	445	1343
0104ac06	dex-1017 - individual photo gallery (#444)	444	1017
8e886383	Social groups UI (#438)	438	762, 1349
127cbdbc	Script for printing unused translation keys (#443)	443	N/A
2b14d53c	dex 1371 - remove data manager role (#442)	442	1371
ab9a4738	DEX-1176 add minWidth (#434)	434	1176
155b6969	dex 1082 - sighting queue and progress metrics (#432)	432	1082
a267668f	Update version to 1.0.2 (#436)	436	N/A
06dda193	922 follow up (#435)	435	DEX-922

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "lint": "NODE_ENV=development BABEL_ENV=development ./node_modules/.bin/eslint --fix \"src/**/*.{js,jsx}\"",
     "format": "./node_modules/.bin/prettier --write \"src/**/*.{js,jsx}\"",
     "test": "echo \"Error: no test specified\" && exit 1",
+    "print-unused-keys": "node ./scripts/printUnusedKeys.js",
```
Comment (Atticus29) [inline package.json:13]: Why did we add this? What does this do?

File: scripts/printUnusedKeys.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+async function printUnusedKeys() {
+  /* eslint-disable */
+  for await (const translationKey of translationKeys) {
+    const results = await find(translationKey, './src');
+    const matches = Object.values(results);
+    const trueMatches = matches.filter(match => {
+      const matchingLines = match?.line || [];
+      const linesWithSingleQuoteMatch = getLinesWithMatch(
```
Comment (Atticus29) [inline scripts/printUnusedKeys.js:18]: These seem to be mislabeled?

File: src/components/dataDisplays/cellRenderers/SocialGroupRoleRenderer.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  const roles = get(data, ['social_group_roles', 'value'], []);
+  const matchingRole = roles.find(role => role?.guid === value);
+  const knownRole = value && matchingRole?.label;
+  const label =
+    knownRole || intl.formatMessage({ id: 'UNKNOWN_ROLE' });
+
+  const CoreComponent = <CoreForwardRef value={label} {...rest} />;
+
+  return noWrap ? (
+    <OverflowController title={label}>
```
Comment (Atticus29) [inline src/components/dataDisplays/cellRenderers/SocialGroupRoleRenderer.jsx:36]: I still want to ask you about this, @Emily-Ke

File: src/components/dialogs/SinglePointDialog.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
Comment (Atticus29) [inline src/components/dialogs/SinglePointDialog.jsx:1]: I don't understand what this file is or does by its name.

File: src/components/fields/edit/LatLongEditor.jsx [hunk 5]
```diff
@@ -37,34 +107,44 @@ export default function LatLongEditor({
...
+  useEffect(() => {
+    if (setterType === setterTypes.internal) {
+      const latitude =
+        latitudeString === '' ? null : parseFloat(latitudeString);
+      const longitude =
+        longitudeString === '' ? null : parseFloat(longitudeString);
+      onChange([latitude, longitude]);
     }
-  }, [get(savedMapLatLng, '0'), get(savedMapLatLng, '1')]);
+  }, [latitudeString, longitudeString, setterType]); // onChange is intentionally excluded.
```
Comment (Atticus29) [inline src/components/fields/edit/LatLongEditor.jsx:138]: Leaving these notes on useEffects will be super handy, I think.

File: src/models/site/useSiteSettings.js [hunk 6]
```diff
@@ -1,66 +1,9 @@
...
-     * changed for some reason, extensive QA of the RegionEditor component
-     * will be necesssary. */
-    data = merge(configuration, schemaData);
-  }
-
-  return useMemo(
-    () => ({ data, loading, error, siteSettingsVersion }),
-    [data, loading, error, siteSettingsVersion],
-  );
+  return useFetch({
```
Comment (Atticus29) [inline src/models/site/useSiteSettings.js:5]: I don't understand why this file change was necessary.

File: src/pages/reportSighting/ReportForm.jsx [hunk 7]
```diff
@@ -277,21 +277,68 @@ export default function ReportForm({
...
+              ]);
+
+              const isTimeSpecificityDefault =
+                formTimeSpecificity === defaultTimeSpecificity;
+
+              const isTimeDefault = formTime === defaultTime;
+
+              if (
+                !formTimeSpecificity ||
+                isTimeSpecificityDefault ||
```
Comment (Atticus29) [inline src/pages/reportSighting/ReportForm.jsx:326]: Why would default result in incomplete time specificity here?

**Review by Emily-Ke — APPROVED** [review]

File: src/pages/userManagement/utils.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
+export function formatUserMessage(user, intl) {
```
Comment (Atticus29) [inline src/pages/userManagement/utils.js:1]: I think this file should have a more illustrative name than utils.js. I don't love relying on directory structure for context.

**Pull request merged**